modules/auxiliary/scanner/http/http_version.rb

```diff
@@ -26,14 +26,28 @@ def initialize
     register_wmap_options({
         'OrderID' => 0,
         'Require' => {},
-      })
+    })
+
+    register_options([
+      OptString.new('TARGETURI', [true, 'The URI to use', '/'])
+    ])
+
+    register_advanced_options([
+      OptString.new('HTTP_METHOD', [true, 'HTTP Method to use', 'GET']),
```

Comment:
You are right. I was checking that and the random method is working well. Then the update of "METHOD" in this module has no sense. But I was reading evasion options in  https://github.com/rapid7/metasploit-framework/wiki/How-to-send-an-HTTP-request-using-Rex::Proto::Http::Client and I see that 'version_random_case', 'version_random_valid' and 'version_random_case' has N/A. Why? It could be a another way of evasion.